### DISCUSSION TITLE
How are you using adamspy?

### DISCUSSION THREAD

**Discussion by bthornton191** [bthornton191/adamspy#15] [General]
I know this package is being used because I can see the download traffic. However, I have no idea which features you all are finding useful. I would be grateful if anyone could let me know what features you are using.  That way I could focus on developing those features.

Thanks! 🙏

**Comment by brychvoj** [comment]
Hello,

is there a sumup of what this package does? 

Thanks a lot for your feedback.

Sincerely,
Vojtech

**Reply by bthornton191** [reply]
Not really. I have some auto generated documentation [here](https://bthornton191.github.io/adamspy/#). Part of the reason I started the discussion is because I would like to put together some better documentation.

But in short, it is a collection of tools for working with data from the multi-body dynamics software package [Adams](https://hexagon.com/products/product-groups/computer-aided-engineering-software/adams?). Are you an Adams user?

**Reply by brychvoj** [reply]
Hello, thanks for the reply!

Yes, I'm an Adams user. I work in the commecial vehicles industry and I use A/View and more often A/CAR for small subsystem's dynamic behavior (axles, steering components, powertrain), full vehicle simulation (ride & handling), load generation etc.

I came across your package by accident so I was wondering what it does as it was not clear to me from the documentation.
